pyproject.toml

```diff
@@ -50,7 +50,7 @@ requires = [
     "setuptools >=61",
     "setuptools_scm[toml] >=3.4",
     "wheel",
-    "oldest-supported-numpy",
+    "numpy>=2.0.0b1",
```

Comment:
@mcara I was able to get this to work by forcing it to build with the newest build of Numpy 2.0 (`2.0.0b1`). This works because the 2.0 API is backwards compatible (according to the changelog), so it's able to be run with Numpy 1.*

When Numpy 2.0 is released, I'll update this to be `>=2.0.0`